governance/reviewers-maintainers.md

```diff
@@ -1,40 +1,19 @@
-Unlike other Special Interest Groups (SIGs), the O3DE Documentation and Community SIG maintains two reviewer roles requiring SIG approval: *Documentation reviewer* and *technical reviewer*. Learn how to become either, and file a [nomination issue](https://github.com/o3de/sig-docs-community/issues/new?assignees=&labels=&template=reviewer-maintainer-nomination.md&title=SIG+Reviewer%2FMaintainer+Nomination) to become an official reviewer or maintainer!
+## Documentation and Community reviewer roles
 
-For more on the Reviewer and Maintainer roles, read our [Community Membership](https://github.com/o3de/community/blob/main/community-membership.md) information. 
+The Documentation and Community SIG maintains the following reviewer pools:
 
-----
-<!-- CUT ABOVE THIS LINE BEFORE POSTING -->
-### Nomination Guidelines
+* **Documentation Reviewer** - Responsible for reviewing documentation content to `o3de.org`, or conducting reviews of certain `o3de` in-product documentation elements.
+* **Community Reviewer** - Responsible for reviewing community materials, including contributor guides and blogs.
 
-#### Documentation reviewer
+We also have maintainers, as other SIGs do. Documentation and Community maintainers are reviewers who have been elevated from the documentation or community
+reviewer pools to take on additional responsibility, and serve as trusted sources of guidance.
 
-_Documentation reviewers_ review for conformance with the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/), appropriateness of content, and other subjective factors to keep documentation content aligned. 
+### Technical reviewer roles
 
-To become a documentation reviewer, you must have:
+The Documentation and Community SIG also maintains a technical reviewer pool for the `o3de.org` website and its related infrastructure.
 
-* 6+ documentation contributions merged to o3de.org or O3DE
-* 2+ Reviewer or Maintainers of the Documentation and Community SIG who support the nomination.
+The technical SIGs also maintain reviewer pools for becoming a technical reviewer. 
```

Comment:
I think it would be helpful to differentiate this kind of a technical reviewer vs. a docs technical reviewer. You can link to the community's guidelines on the roles: https://github.com/o3de/community/blob/main/community-membership.md